assets/aws/files/bin/teleport-generate-config

```diff
@@ -161,8 +161,25 @@ EOF
 
     # if we are using letsencrypt (i.e. not ACM) then append config lines
     if [[ "${USE_ACM}" != "true" ]]; then
-        echo "  https_cert_file: /var/lib/teleport/fullchain.pem" >> /etc/teleport.yaml
-        echo "  https_key_file: /var/lib/teleport/privkey.pem" >> /etc/teleport.yaml
+        echo "  https_keypairs:" >> /etc/teleport.yaml
+        echo "    - cert_file: /var/lib/teleport/fullchain.pem" >> /etc/teleport.yaml
+        echo "      key_file: /var/lib/teleport/privkey.pem" >> /etc/teleport.yaml
```

Comment:
nit: can this be a single `echo` command with multiline string?
same below

Reply:
It could; I didn't do that previously because it made it harder to get the indentation exactly right.